source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to use **collations**, a set of sorting rules, to
+run operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+The order in which they would appear using the default *binary
+collation* is:
+
+::
+
+   cote
+   coté
+   côte
+   côté
+
+If we apply the Canadian French collation, the order is:
+
+::
+
+   cote
+   côte
+   coté
+   côté
+
+.. note::
+
+   Collations are available in MongoDB 3.4 and later.
```

Comment:
This note feels like it might be more suited to the very top of the page. (beneath "on this page"). What do you think?

Reply:
I thought of placing it at the top as well, but it seemed too early on in the document to make an admonition before describing what a collation is. What if I just used some italic text, similar to this pattern used in the server manual: https://docs.mongodb.com/manual/tutorial/configure-encryption/

Reply:
Instead of "New in version 3.2." it could read "Collations are available in MongoDB 3.4 and later" to prevent confusion with the driver version. The top of the configure-encryption page is also a mess.

Reply:
I like this solution a lot!